content/hi/docs/contribute/new-content/new-features.md

```diff
@@ -60,7 +60,7 @@ card:
 2. [कुबेरनेट्स/कुबेरनेट्स](https://github.com/kubernetes/kubernetes) PR(s) और [कुबेरनेट्स/एन्हांसमेंट्स](https://github.com/kubernetes/enhancements) मुद्दों के लिंक शामिल करने के लिए पुल अनुरोध विवरण संपादित करें।
```

Comment:
I don't know if we should be localizing pull request here.

Reply:
I think also as this is name of our repository, let's see what others are thinking about this till now I will rebase all these commits.